pkgs/applications/misc/kjv/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  name = "kjv";
+  version = "20190311";
```

Comment:
please use `unstable-YYYY-MM-DD`. See https://nixos.org/nixpkgs/manual/#sec-package-naming

Reply:
there should be a link to that section on the contributing doc....